hack/remove-calico-policy/README.md

```diff
@@ -0,0 +1,13 @@
+# Disabling and removing Calico policy
+
+1. Stop and remove the calico-node Pod on each Node to prevent network policy from being (re)programmed.
+    `kubectl delete daemonset calico-node -n kube-system`
+
+2. Create a configmap with the bash script to remove the Calico iptables rules (this configmap will be used in the DaemonSet in the following step).
+    `kubectl create configmap remove-calico-policy-config --namespace=kube-system --from-file=/path/to/remove-calico-policy.sh`
+
+3. Remove all Calico iptables chains by running the following on each Node (e.g. using a privileged DaemonSet with host networking) or on a specific problem Node:
```

Comment:
I think this last bullet isn't right - it won't do this for a specific problem node if you install the DaemonSet.

Let's organize this document like so:

```
# Disabling and removing Calico policy
<introduction content here>

### Requirements
<minimum calico version, etc>

### Removing policy on a single node
1. Stop calico on the node
2. Run the provided script on that specific Node (either by hand or using nodeSelector)

### Removing policy on all nodes
1. Stop calico on all nodes
2. Create ConfigMap
3. Run DaemonSet
```